travis/setup_xspec_ds9.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Tarballs to fetch
+ds9_site=http://ds9.si.edu/download/centos6/;
+xpa_site=http://ds9.si.edu/download/centos6/;
+ds9_tar=ds9.centos6.7.5.tar.gz;
+xpa_tar=xpa.centos6.2.1.17.tar.gz;
+
+# Fetch them
+wget --quiet $ds9_site$ds9_tar;
+wget --quiet $xpa_site$xpa_tar;
+
+# untar them
+THIS_DIR=$(pwd);
+cd $MINICONDA/bin; tar xf $THIS_DIR/$ds9_tar; tar xf $THIS_DIR/$xpa_tar; cd -;
```

Comment:
I'm guessing the travis failures are because MINICONDA needs to be changed to miniconda in this file?

Reply:
Also, now this is a separate script, this need not all be on one line.

Reply:
Yeah, my IDE can be smart at refactoring, but I need to be smarter ;)